modules/ossm-accessing-bookinfo-application-using-istio-gateway-injection.adoc

```diff
@@ -88,15 +90,16 @@ metadata:
     istio: ingressgateway
     release: istio
   name: ingressgatewaypdb
-  namespace: istio-ingress
+  namespace: bookinfo
 spec:
-  minAvailable: 1
+  minAvailable: 1 <1>
   selector:
     matchLabels:
       istio: ingressgateway
 ----
+<1> This example ensures one replica is running if a pod gets restarted on a new node.
 
-. Obtain the gateway host name and the URL for the product page:
+. Obtain the gateway host name and the URL for the product page by running the following command:
 +
 [source,terminal]
```

Comment:
While you are in here: this is two commands in one block. [We should at most have one per block](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#single-command-per-code-block).

Reply:
done

Reply:
thank you!